.htaccess

```diff
@@ -28,6 +28,12 @@
     php_flag expose_php Off
 </IfModule>
 
+### URL Helper ###
+
+# Removes any additional unwanted segments added to URL
+<IfModule mod_rewrite.c>
+    RewriteRule ^index\.php/module/action/param1/.*/(.*+)$ /$1 [L,R=301]
```

Comment:
When does this occur? pH7CMS doesn't have these URL patterns. And certainly not `action/param1/`
Do you mind giving me a hint on this one? I don't really see why it would be useful.

Reply:
When we set the cache to on we get this long URL error 

```https://example.com/index.php/module/action/param1/static/PFBC/js/jquery/rss/static/css/js/jquery/smoothness/templates/themes/datelove/img/icon/xml/sitemap/user/help/rss/legal/rss/comment-picture```

This seems to be a problem that a lot of people are having as I have answered many issues on this problem. We have spent days trying to find the problem in the code and have even had the hosting company look into it. They said its due to a cache problem but couldn't pinpoint the exact cause. 

The rule added removes the added segments if they are there and is ignored if they are not.

Reply:
Think I found the problem...

```/public_html/_protected/framework/Mvc/Router/Uri.class.php```

Reply:
Ok after extensive testing this is the problem... The file mentioned above controls the sitemap cache, at the very top there is a warning that says if you are on shared hosting you may want to disable this feature, this is because most hosting companies restrict you on inode usage and they allow the files to become overaged. This confuses the script responsible for the sitemap cache "

```/***/***/hidden/_protected/framework/Mvc/Router/Uri.class.php"```

What is the solution? I think the software should ship with this disabled as a majority of users are on shared hosting and I will add it to the cache settings menu when I have time, unless you wish to do it?